RCAIDE/Library/Methods/Weights/Correlation_Buildups/General_Aviation/compute_operating_empty_weight.py

```diff
@@ -283,29 +282,42 @@ def compute_operating_empty_weight(vehicle, settings=None):
         S_fus       = fuselage.areas.wetted
         diff_p_fus  = fuselage.differential_pressure
         w_fus       = fuselage.width
-        h_fus       = fuselage.heights.maximum
-        l_fus       = fuselage.lengths.structure
+        h_fus       = fuselage.heights.maximum 
+        l_fus       = fuselage.lengths.total-fuselage.lengths.tail  
```

Comment:
Just a quick question, is it supposed to be only the tail length?

Reply:
I am not 100% sure , but I think so

Reply:
Taking a look at Raymer I don't think it is only tail length. I believe this is supposed to be the structural length of the fuselage (length minus radome and tail cone). The tail length variable is l_w2h and is given separately.